conkit/core/contactmap.py

```diff
@@ -894,6 +918,72 @@ def remove_neighbors(self, min_distance=5, max_distance=sys.maxsize, inplace=Fal
                 contact_map.remove(tuple(contactid))
         return contact_map
 
+    def trim_region(self, region, inplace=False):
+        """Trim away a specific region of a contact map
+
+        Parameters
+        ----------
+        region : int, list, tuple, required
```

Comment:
required adds no value to this docstring. you're defining it as an argument in the function signature, it is therefore automatically required